include/soci/blob.h

```diff
@@ -50,10 +51,10 @@ class SOCI_DECL blob
 
     void trim(std::size_t newLen);
 
-    details::blob_backend * get_backend() { return backEnd_; }
+    cxx_details::shared_ptr<details::blob_backend> get_backend() { return backEnd_; }
```

Comment:
Do we really have to change this? It's a backwards-incompatible change and it seems like it could be avoided by keeping the existing return type here and adding some new `get_backend_ptr()` if it's needed.

Reply:
It is a good suggestion indeed.

If we really need to keep this around, I will have this in mind. However, if we normalize the use of `SOCI` for `blob` data, as explained on the [issue 922](https://github.com/SOCI/soci/issues/922), and separate `class blob;` from `class blob_backend;` neither the `shared_ptr` neither this function will be really necessary anymore.